fix simple typo

lpmi-13 opened the pull request.

sim590 commented:
Thank you for your contribution @lpmi-13! I'm pleased to see interest in this project even for a small contribution like this :wink: and also since we have not made lots of contribution ourselves in the recent past.

Would you mind rewording your commit message to

```
readme: fix double "the" typo
```

It is not specified anywhere yet, but since the last time we have contributed, @ablondin and I use some convention along the lines of this:

https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html

and also require that the name of the file or component prefixes the commit message, so in this case it is `readme`.

I hope you don't mind doing that, so I can simply press the "merge" button. I wouldn't mind doing it myself, but I prefer asking the contributor so I don't modify their contribution without notice and the "merge" button can be pressed.

I will write the requirements in a file somewhere.

Thanks!

lpmi-13 commented:
no problem. I've updated the commit message as requested, and also added a second commit with some basic requirements you mentioned into a CONTRIBUTING file, if it seems appropriate.
